src/main/java/org/killbill/billing/plugin/payment/dao/gen/tables/records/TestpaymentHppRequestsRecord.java

```diff
@@ -408,7 +406,55 @@ public TestpaymentHppRequestsRecord values(ULong value1, String value2, String v
         return this;
     }
 
-    // -------------------------------------------------------------------------
+
+	@Override
+	public ULong component1() {
```

Comment:
Did you regenerate these files per the tool? Everything under the `gen` package is autogenerated.

Check https://github.com/killbill/killbill-stripe-plugin/blob/master/src/main/resources/README.txt for instructions.

Reply:
I did now.